config/routes_into_teaching.yml

```
@@ -91,11 +91,11 @@ routes:
 
   - title: Teacher degree apprenticeship
     course_length: 4 years
-    course_fee: No - you will be paid a salary
-    funding: "No"
-    description: These courses are in high demand and very competitive, so it’s important to apply as soon as you can if you’re eligible.
+    course_fee: "No"
+    funding: These courses pay a salary. Bursaries, scholarships and student loans are not available. 
+    description: The teacher degree apprenticeship (TDA) is a teacher training programme where you’ll work in a school and earn a salary while getting a bachelor’s degree and qualified teacher status (QTS). The teacher training providers confirmed so far offer TDAs in secondary maths. TDAs in other subjects will be added throughout the year, up until September 2025, with training due to start in autumn 2025.
```

```suggestion
    description: The teacher degree apprenticeship (TDA) is a teacher training programme where you’ll work in a school and earn a salary while getting a bachelor’s degree and qualified teacher status (QTS). The teacher training providers confirmed so far offer TDAs in secondary maths. TDAs in other subjects will be added throughout the year, with training due to start in autumn 2025.
```
Do we need the up to September bit?

---

We might need to check with Sarah on this one, as I wasn't in the convos with policy so it might've been a request from them?

---

Resolving here - will address separately with @Sarah-DfE